tests/NgTests/Data/TestCatalogEntries.resx

```diff
@@ -879,11 +879,11 @@
   "lastEdited": "2015-01-01T00:00:00Z",
   "licenseNames": "",
   "licenseReportUrl": "",
-  "listed": true,
+  "listed": false,
   "packageHash": "0LUKl3HqBsrLxjTaA+J6vaa+XB5zWb5hTJb1Ht74oIzPsBJmZalApZAIdueMgzbsUxG74jjfnDZWoxUrs2A3dg==",
   "packageHashAlgorithm": "SHA512",
   "packageSize": 2540,
-  "published": "1900-01-01T01:00:00+01:00",
+  "published": "1900-01-01T01:00:00+00:00",
```

Comment:
Don't we want to include a test case with a `DateTime` with a negative offset to verify this issue?